tests/system/Autoloader/FileLocatorTest.php

```diff
@@ -103,7 +103,10 @@ public function testLocateFileWithFolderNameInFile(): void
     {
         $file = '\App\Views/errors/html/error_404.php';
 
-        $expected = APPPATH . 'Views/errors/html/error_404.php';
+        $expected = APPPATH . 'Views'
+            . DIRECTORY_SEPARATOR . 'errors'
+            . DIRECTORY_SEPARATOR . 'html'
+            . DIRECTORY_SEPARATOR . 'error_404.php';
```

Comment:
To be honest, I don't like to write test code like this.
Because it is a bit more difficult to read.
So, I would like a different solution.

Reply:
@kenjis In my case, I usually prefer to use `implode()` for paths, which I find improves clarity and maintainability. Here's an example of how it could be refactored using `implode()`
```php
$expected = implode(DIRECTORY_SEPARATOR, [APPPATH, 'Views', 'errors', 'html', 'error_404.php']);
```